libs/wire-api/src/Wire/API/Routes/Public/Spar.hs

```diff
@@ -203,7 +204,10 @@ sparResponseURI (Just tid) =
 
 type APIScim =
   OmitDocs :> "v2" :> ScimSiteAPI SparTag
-    :<|> "auth-tokens" :> APIScimToken
+    :<|> "auth-tokens"
+      :> CanThrow PasswordAuthenticationFailed
```

Comment:
that was missing before, right?  interesting.  looking forward to polysemizing `CanThrow` away...  :)

Reply:
[for another PR] that `OmitDocs` above btw may make the `CanThrow` pretty pointless, since it makes sure this doesn't make it into the swagger docs.  we should probably remove `OmitDocs` entirely and have all the scim docs online as well.  the reason we didn't was that there is a standard for it available where the information exists, and we didn't want to write all the schema code.